src/app/components/Application/Main.jsx

```diff
@@ -238,6 +238,7 @@ Main.propTypes = {
   listOptions: PropTypes.object,
   currentSeason: PropTypes.string,
   currentAudience: PropTypes.string,
+  location: PropTypes.shape(PropTypes.object),
```

Comment:
If we use `PropTypes.shape` here,  we should indicate the prop types of its children as the argument. Such as

```
 {
   hash: PropTypes.string,
}
```